web/sites/default/services.yml

```diff
@@ -0,0 +1,174 @@
+parameters:
```

Comment:
It's better not to commit this file for the sake of our project:
1. If I merge this in, I'd actually be enabling debugging for production as well
2. The sites/default folder has restrictive permissions, so team members would get a 'permission denied' issue when merging this PR in locally. That does not mean we should never commit changes in this folder, but it's worth being aware of.

The better option is to edit the .gitignore file in the root of the project and to add a line for services.yml